frontend/src/libs/components/app/app.tsx

```diff
@@ -13,10 +15,18 @@ const App: React.FC = () => {
     authenticatedUserDataStatus: auth.authenticatedUserDataStatus,
   }));
 
-  useEffect(() => {
-    void dispatch(authActions.getAuthenticatedUser());
+  const handleGetAuthenticatedUser = useCallback(async (): Promise<void> => {
+    const hasToken = await storage.has(StorageKey.TOKEN);
```

Comment:
let's do this in getAuthenticatedUser action directly and return null if there is no token, we can't use services inside components